### PR TITLE
Fix all linter errors

### PR DESCRIPTION
**Pull request by billfeng**
The linters are now all passing

### CODE REVIEW

File: server/scripts/location.js [hunk 1]
```diff
@@ -34,10 +33,10 @@ const stations = ['Nishi-Funabashi', 'Yoyogi-Uehara', 'Naka-Meguro', 'Wako-shi',
 
     fs.writeFile(LOCATION_DATA_PATH + DATA_FILE, JSON.stringify(results), (err) => {
       if (err) throw err;
-      console.log('The file has been saved!');
+      // console.error('The file has been saved!');
     });
```
Comment (kumilange) [inline server/scripts/location.js]: @bill-feng Thanks for noticing this 👍   Can you remove this comment out?

File: server/scripts/location.js [hunk 2]
```diff
@@ -34,10 +33,10 @@ const stations = ['Nishi-Funabashi', 'Yoyogi-Uehara', 'Naka-Meguro', 'Wako-shi',
...
 
   } catch (err) {
-    console.error('Error updating records', err);
+    // console.error('Error updating records', err);
```
Comment (kumilange) [inline server/scripts/location.js]: @bill-feng Thanks for noticing this 👍   Can you remove this comment out?

**Comment by kumilange** [comment]
@bill-feng I left 2 minor comments on comment out. Other than that, LGTM! 👍

**Review by avantgardnerio — APPROVED** [review]
Errors were being swallowed. Bill & I fixed.

LGTM

**Pull request merged**